### ISSUE TITLE
A11y_ML .NET Model Builder_ Object detection-Data Preview_ Screen Reader: Narrator/NVDA are not reading the proper name when focus is on images which are uploaded

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1159]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev2; #Benchmark; #Visual Studio; #Screen Reade; #Narrator; #NVDA; #WCAG1.3.1; #AILimited;

### Environment Details:
Microsoft Visual Studio 2019 Preview Version 16.8.0 Preview 6.0

### Repro Steps:

1.Start Screen Reader
2.Open Visual Studio Enterprise 2019 Preview 
3. Create a new .Net Core Console App (C#) 
4. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
5. Tab till "Object Detection" scenario and select the scenario
6. Tab till "Set up work Space" button and press enter
7. Setup a workspace and tab till "Next Step" button and press enter
8. Upload the file and navigate till data preview section and verify how screen readers are reading the images that are uploaded

### Actual Result:
Narrator/NVDA are not reading the proper name when focus is on images which are uploaded. When focus comes to the first image screen readers are just reading as "image" and when focus comes to image controls screen readers are reading the  path of uploaded image instead of reading the name

### Expected Result:
Narrator/NVDA should read the proper name of the images

### User Impact:
Screen Reader users will not get to know the what are those images if screen reader doesn't read the name for the images properly

### Attachment for Reference

[1159_ML .NET_Narrator not reading the name of the image.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5686961/1159_ML.NET_Narrator.not.reading.the.name.of.the.image.pptx)
[1159_ML .NET_NVDA not reading the name of the image.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5686975/1159_ML.NET_NVDA.not.reading.the.name.of.the.image.pptx)

**Comment by beccamc** [comment]
Build https://devdiv.visualstudio.com/_apis/resources/Containers/7343454/VSSetup?itemPath=VSSetup%2FMicrosoft.ML.ModelBuilder.vsix

**Comment by SaisreeDandamudi** [comment]
Verified the bug in latest build and the issue is fixed. Hence closing the bug
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.10.0 Preview 1.0
[1159_ML.NET_Fixed_Narrator.not.reading.the.name.of.the.image.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/6141303/1159_ML.NET_Fixed_Narrator.not.reading.the.name.of.the.image.pptx)

**Comment by poojapatil36** [comment]
Github tag: #Closed;
